### PR TITLE
add google calendar, Calendar link to main nav bar

### PR DESCRIPTION
**Pull request by hollomancer**

### CODE REVIEW

**Comment by hollomancer** [comment]
#149 - still needs the link or form to request an item be added to the calendar.

**Comment by rickr** [comment]
Is this is iframe code from #148?

Can you create a new issue in the calendar milestone to add a link for a new calendar date?

**Comment by hollomancer** [comment]
Yeah, it is. Added new issue at #170 - I don't have permissions to add to milestone though.

**Comment by jjhampton** [comment]
@hollomancer I actually had started working on #149 yesterday (which is why I self-assigned it)- looks like we both added both some similar code - I'd pushed up my stuff to a forked repo/branch here: https://github.com/jjhampton/operationcode/tree/JJH/add-calendar/issue-149

For now, we can merge your work in and then I can rebase on it - I also had some additional CSS styles and HTML attributes in the iframe that made it viewable on mobile that I'd like to add.

@rickr @davidmolina I think it would be a good idea to explore how to allow more people to assign themselves to issues - currently only members of the Operation Code org can do this.

**Comment by jjhampton** [comment]
Tagged #170 w/ the 'calendar' milestone.

File: app/views/pages/calendar.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="container bs-docs-container">
+  <div class="row">
+    <div class="col-md-12" role="main">
+        <h1 id="download" class="page-header">Here's what's coming up.</h1>
```
Comment (jjhampton) [inline app/views/pages/calendar.html.erb]: What's the reason for the `id="download"` here?

**Comment by jjhampton** [comment]
Merging this in, but leaving #149 open for now. Will add my PR onto @hollomancer's work and reference the same issue.

**Pull request merged**